feature-libs/user/account/root/model/otp-login.model.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface LoginForm {
+  purpose: string;
```

Comment:
I wanna to say, it should hardcode in somewhere as hte loginForm only used for login

Reply:
Ah, I got, it seems that here is a **name**  misunderstand, LoginForm model is used as payload to call VerificationToken API, instead of otp login html, I have renamed as **VerificationTokenCreation** model to have a correct understanding.  This is a good point, thanks.